Source/UZKFileInfo.m

```diff
@@ -51,6 +82,22 @@ - (instancetype)initWithFileInfo:(unz_file_info64 *)fileInfo filename:(NSString
 }
 
 
+#pragma mark - Private Class Methods
```

Comment:
I'd like this section to be between `Properties` and `Private Methods`, if you could please move it down.

Reply:
Let me understand what you mean correctly. Do you mean moving "itemIsDirectory" and "itemIsSymbolicLink" into the "#pragma mark - Properties" code block?

Reply:
I mean moving lines 85-100 down to where line 114 is. So the sections would go in the order:

# Properties
# Private Class Methods
# Private Methods

I don't usually separate class methods from instance methods, but it's not a bad idea here. I just want all the private methods together.